CI/integration_tests/calculators/test_angular_distribution_function.py

```diff
@@ -69,3 +70,21 @@ def test_project(traj_file, true_values, tmp_path):
         computation["NaCl"].data_dict[item].pop("max_peak")
 
     assertDeepAlmostEqual(computation["NaCl"].data_dict, true_values, decimal=-2)
+
+
+def test_low_memory(traj_file, true_values, tmp_path):
+    """Test the ADF called from the project class"""
+    mds.config.memory_fraction = compute_memory_fraction(0.1)
+    os.chdir(tmp_path)
+    project = mds.Project()
+    project.add_experiment(
+        "NaCl", simulation_data=traj_file, timestep=0.002, temperature=1400
+    )
+
+    computation = project.run.AngularDistributionFunction(plot=False)
+
+    for item in computation["NaCl"].data_dict:
+        computation["NaCl"].data_dict[item].pop("max_peak")
```

Comment:
We should not use `.data_dict` anymore.
Why do you remove the `max_peak`?

Reply:
This was a long time ago. Most of the data we have saved is wrong and we need proper test data to check it. In the water data we kind of have this but as of yet, there is no nice thing we can use here. When these calculators were fixed a few months ago this was changed. I think removing data dict is in a different issue somewhere, I did not add this manually.

Reply:
You should be able to replace it with `computation["NaCl"][item].pop("max_peak")`

Reply:
I think we should move it all over at one time but I will be trying to add proper data for several tests next week and I will then re-write many of these.